CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+# Contributing to mockrobiota
+
+[mockrobiota](https://github.com/caporaso-lab/mockrobiota) is an open-source, public resource, and we depend on community commitment to grow and evolve. Thank you for sharing your awesome mock communities with the world.
```

Comment:
*open-source, public resource* is awkward, how about:

*mockrobiota is a public resource that depends on community involvement to grow and evolve.*